Assets/ItemButton.cs

```diff
@@ -0,0 +1,15 @@
+using TMPro;
+using UnityEngine;
+
+public class ItemButton : MonoBehaviour
+{
+	public TMP_Text _itemBalanceText;
```

Comment:
[SerializeField] private
unity serialized fields should never be public because it ruins encapsulation and can lead to unexpected behaviour/
if you need to access propertyy from outside you should create get only property